cmd/create.go

```diff
@@ -24,23 +24,28 @@ import (
 )
 
 // createCmd represents the create command
```

Comment:
How about rephrasing this to:

```suggestion
// createCmd generates the project using cookiecutter
```

Reply:
Fixed in e2cf98e3c0ddd8281cca1f29aca709577fdbaee4. Actually that comment was generated by Cobra.